src/hooks/usePrevious.ts

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export function usePrevious<Type>(
```

Comment:
Nice!

Do people still argue about `T` vs using something else for generics? `Type` reads more friendly IMO 🤷‍♂️

Reply:
Yeah there was a bit of a commotion about that a while back. I think single letter generics make it more difficult to understand, especially for people who are newer to Typescript.

It's the same as using single letter variables/args, but with an extra layer or two of complexity